kasa/smart/smartdevice.py

```diff
@@ -309,6 +308,26 @@ async def _initialize_features(self):
             for feat in module._module_features.values():
                 self._add_feature(feat)
 
+    def has_module(self, module_name):
```

Comment:
Is there a need for a separate method for checking the existence vs. checking of `get_module` returns non-None value?

Reply:
Done